types/serializers/wallet.d.ts

```diff
@@ -0,0 +1,13 @@
+import { StringSerializer } from './basic';
+import { StructSerializer, VectorSerializer } from './collections';
+
+declare const approvalDelta : StructSerializer<{
+    active_approvals_to_add: VectorSerializer<StringSerializer>,
+    active_approvals_to_remove: VectorSerializer<StringSerializer>,
+    owner_approvals_to_add: VectorSerializer<StringSerializer>,
+    owner_approvals_to_remove: VectorSerializer<StringSerializer>,
+    key_approvals_to_add: VectorSerializer<StringSerializer>,
+    key_approvals_to_remove: VectorSerializer<StringSerializer>,
+}>;
+
+export default approvalDelta;
```

Comment:
Remove `export default` and use `export` (your declaration files should duplicate structure of JS-files)